test/forms/TransportSpec.scala

```diff
@@ -82,12 +82,12 @@ class TransportSpec extends BaseSpec {
 
     "return no error" when {
 
-      "values are correct" in {
+      "values are correct for different country codes" in {
+        val transportPoland = Transport(Sea, "PL")
+        Transport.form().fillAndValidate(transportPoland).errors.length must be(0)
```

Comment:
I think you could change it to 
```suggestion
        Transport.form().fillAndValidate(transportPoland).errors must be(empty)
```